packages/types/src/workflow/cart/add-shipping-method.ts

```diff
@@ -0,0 +1,5 @@
+export interface AddShippingMethodToCartWorkflowDTO {
```

Comment:
question: is this the schema for the input? If so would it make sense to define this more dynamically - e.g., using something like zod? my thinking is that this would make it easier for people to extend and we would still get the type safety.

Reply:
Fair point. I would prefer to have this discussion with the rest of the team. Can we punt this to when everyone is back in office?